node/core/candidate-validation/Cargo.toml

```diff
@@ -19,6 +19,7 @@ polkadot-node-primitives = { path = "../../primitives" }
 polkadot-node-subsystem = {path = "../../subsystem" }
 polkadot-node-subsystem-util = { path = "../../subsystem-util" }
 
+# TODO: Do we need this?
 [target.'cfg(not(any(target_os = "android", target_os = "unknown")))'.dependencies]
```

Comment:
Why is this here? The `polkadot-node-core-pvf` dependency isn't gated like this anywhere else.

Reply:
Can probably be removed.